README.md

```diff
@@ -4,22 +4,56 @@ This is an online multiplayer take on the classic party game Mafia. The game spl
 However, since this is an OSS project, developers are free to edit rules and add new functionality as they please.
 
 <p align="center">
-<img src="https://cdn2.iconfinder.com/data/icons/rounded-white-basic-ui-set-3/139/Photo_Add-RoundedWhite-512.png" height=200; width=200>
+  <img src="https://cdn2.iconfinder.com/data/icons/rounded-white-basic-ui-set-3/139/Photo_Add-RoundedWhite-512.png" height=200; width=200>
 </p>
 
 
 Please review the [documentation](https://github.com/Mafia-Hands/Mafia/wiki) for guidelines and information about the software.
 
 ## Installation
+Before you install & get started with the Mafia project, please you have installed [nodejs](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) on your computer.
 
-Use the package manager [npm](https://www.npmjs.com/get-npm) to install all dependencies.
+This project uses the package manager [npm](https://www.npmjs.com/get-npm) to install all dependencies.
 
+### For frontend
 ```bash
+git clone git@github.com:Mafia-Hands/Mafia.git
+cd Mafia
+cd frontend
 npm install
+npm start
 ```
+
+### For backend
+```bash
+git clone git@github.com:Mafia-Hands/Mafia.git
+cd Mafia
+cd backend
+npm install
+npm start
+```
+
+## Usage
+For development purpose, please check the [Installation](https://github.com/Mafia-Hands/Mafia#installation) and our [contributing guidelines](https://github.com/Mafia-Hands/Mafia/wiki/Contributing-Guidelines).
+
+For playing, please read through the [rules of the Mafia game](https://github.com/Mafia-Hands/Mafia/wiki/Game-Rules) first, even though you have played Mafia game before. Now, let's go through how to create, join, start a game quickly. By the default setting, after you start running the front-end application, you would be able to access the game by accessing localhost:3000.
+
+If you do not have an existing lobbyID, you would have to create the room for you and your friend. To create the lobbdyID, you have to click on the button 'Create new game' then share the LobbyID with your friends who would like to join the game. If you already have the LobbyID from the host, then you just input your nickname and the lobbyID, then click join.
+
+## Support
+If you have encountered any problems when setting up or developing the project, you can create a Github issue and tag the person who wrote the code related to your issues.
+
+## Roadmap
+We do have lists for the tasks that need to be done in the Github projects page.
```

Comment:
Plans for future works can be found on the relevant Wiki page. Additionally, outstanding issues and requests can be tracked and visualised on our Github Projects page.